### PR TITLE
Add missing test.

### PR DESCRIPTION
**Pull request by vasole**
One test was not run in manual mode.

### CODE REVIEW

File: PyMca5/tests/ConfigDictTest.py [hunk 1]
```diff
@@ -196,6 +196,8 @@ def getSuite(auto=True):
         # use a predefined order
         testSuite.addTest(testConfigDict("testConfigDictImport"))
         testSuite.addTest(testConfigDict("testConfigDictIO"))
+        if HAS_H5PY:
```
Comment (woutdenolf) [inline PyMca5/tests/ConfigDictTest.py]: Perhaps use `unitests.skipIf` for this kind of thing?

Comment (vasole) [reply PyMca5/tests/ConfigDictTest.py]: That is the approach used for the automatic testing (it is there in line 125).

It was just missing when running the test standalone.

Comment (woutdenolf) [reply PyMca5/tests/ConfigDictTest.py]: I see, but then why not rely on that and not exclude it explicitly here?

Comment (vasole) [reply PyMca5/tests/ConfigDictTest.py]: Thanks. I misunderstood how it was working.

**Pull request merged**